examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<p>Bit wordy but does explain the zeros. Any chance can truncate the snapshot so that it doesn't look like the wave disappear?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Yeah, I can do that, but still, the first example shouldnt have this problem at all, so I still have some things to figure out.</p>